source/includes/fact-data-prep-oracle-step1.rst

```diff
@@ -4,18 +4,23 @@ instance. Alternatively, you can use an existing Oracle
 service account to connect to Relational Migrator with 
 the appropriate permissions.
 
+.. tip::
+
+   To migrate data from a multi-tenant container database, use a `Common
+   User
+   <https://oracle-base.com/articles/12c/multitenant-manage-users-and-privileges-for-cdb-and-pdb-12cr1>`__.
+
```

Comment:
[question] Why is this a one-step procedure? Shouldn't each substep here just be a step?

Reply:
This is an artifact of removing nested procedures. Since it's the same include used for both snapshot and continuous jobs, the include explicitly uses an ordered list so that it renders as substeps for the longer procedure.

Reply:
Ahhh, got it. Ok!